include/LAGraphX.h

```diff
@@ -777,6 +777,28 @@ int LAGraph_scc (
     char *msg
 ) ;
 
+//****************************************************************************
+LAGRAPHX_PUBLIC
+int LAGraph_RegularReachability // nodes reachable from the starting by the
```

Comment:
Naming.

Reply:
Fixed.

Reply:
Resolving, since discussed verbally Semyon Vyacheslavovich is ok.